### PR TITLE
Fix and update links in Python articles

### LINKED ISSUE #491: Python articles should be interlinked
There are some basic articles on Python in the FCC Wiki. To make them more accessible and indexable, they should be interlinked back and forth. And the central stub for Python should link to those.

### PR DESCRIPTION
**Pull request by alayek**
Closes #491

### CODE REVIEW

**Comment by alayek** [comment]
@Rafase282 @raisedadead please review.

All Python articles on our wiki so far should be interlinked by this PR. I have tested the links locally.

File: Python.md [hunk 1]
```diff
@@ -14,4 +14,16 @@ There are various implementations of Python language available, although most co
 
 Python is so wide-spread in its use, that most Linux and OSX distros come pre-installed with Python. Because a lot of installed software packages use Python internally.
 
-Python is used in data science, machine learning, and especially, web development.
\ No newline at end of file
+Python is used in data science, machine learning, and especially, web development.
+
+To get more familiar with Python, [start here](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Python-Introduction)
+
+If you are looking for something specific Wein Python, pick from this list of resources:
```
Comment (koustuvsinha) [inline Python.md]: "Wein" ? whats that?

**Comment by koustuvsinha** [comment]
looks great @alayek !

**Comment by alayek** [comment]
Thanks :)

**Pull request merged**